src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantImplicitNamespaceImportsDisabled.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantImplicitNamespaceImportsDisabled : SdkTest
+    {
+        public GivenThatWeWantImplicitNamespaceImportsDisabled(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [Fact]
```

Comment:
Does this need to be a WPF app to repro the issue?  If so, then this needs to be a `WindowsOnlyFact`, as it could only be built on Windows.  If the same issue exists with a console app though it would be better to use that.